src/storage/storage.go

```diff
@@ -130,16 +151,22 @@ func (i *TxIterator) Next() *types.Transaction {
 	if !i.rows.Next() {
 		return nil
 	}
+
 	var txidBytes []byte
 	var firstSeen int64
-	if err := i.rows.Scan(&txidBytes, &firstSeen); err != nil {
-		panic(err)
+	var fee uint64
+	var weight int
+	if err := i.rows.Scan(&txidBytes, &firstSeen, &fee, &weight); err != nil {
+		panic(err) // TODO: cleanup
```

Comment:
IMO `panic()` is fine here since the caller can only propagate anyway

Reply:
Agree, will remove it.